csunplugged/templates/topics/topic.html

```diff
@@ -39,13 +39,6 @@ <h2 id="integrations" class="heading-underline">Curriculum Integrations</h2>
     <p>No curriculum integrations for {{ topic }}.</p>
   {% endif %}
 
-  <h2 id="programming" class="heading-underline">Programming Challenges</h2>
-  {% if programming_exercises %}
-    {% include "topics/programming-exercises-table.html" %}
-  {% else %}
-    <p>No programming challenges for {{ topic }}.</p>
-  {% endif %}
-
```

Comment:
Yes, as these are only relevant to lessons (don't have a number related to a topic). This also means in the long term, you could use programming exercises across topics.